services/web/server/src/simcore_service_webserver/licenses/_licensed_items_repository.py

```diff
@@ -204,49 +158,20 @@ async def get(
         return LicensedItemDB.model_validate(row)
 
 
-async def get_by_resource_identifier(
-    app: web.Application,
-    connection: AsyncConnection | None = None,
-    *,
-    licensed_resource_name: str,
-    licensed_resource_type: LicensedResourceType,
-) -> LicensedItemDB:
-    select_query = select(*_SELECTION_ARGS).where(
-        (licensed_items.c.licensed_resource_name == licensed_resource_name)
-        & (licensed_items.c.licensed_resource_type == licensed_resource_type)
-    )
-
-    async with pass_or_acquire_connection(get_asyncpg_engine(app), connection) as conn:
-        result = await conn.execute(select_query)
-        row = result.one_or_none()
-        if row is None:
-            raise LicensedItemNotFoundError(
-                licensed_item_id="Unkown",
-                licensed_resource_name=licensed_resource_name,
-                licensed_resource_type=licensed_resource_type,
-            )
-        return LicensedItemDB.model_validate(row)
-
-
 async def update(
     app: web.Application,
     connection: AsyncConnection | None = None,
     *,
     product_name: ProductName,
     licensed_item_id: LicensedItemID,
-    updates: LicensedItemUpdateDB,
+    updates: LicensedItemPatchDB,
 ) -> LicensedItemDB:
     # NOTE: at least 'touch' if updated_values is empty
     _updates = {
         **updates.model_dump(exclude_unset=True),
         licensed_items.c.modified.name: func.now(),
     }
 
```

Comment:
Basically all the trashing functionality has been removed ... 

The idea here was that when the syncer notices that one of the licensed-resources is not anymore available in itis-vip, it marks it a `trashed` 

If we make local copies, i guess this is not that critical but it is an edge case we should consider. 

And also shame i wasted my time implementing it :-(